pydis_site/apps/content/resources/guides/python-guides/windows/installing-and-using-python.md

```diff
@@ -0,0 +1,236 @@
+---
+title: Installing and Using Python on Windows
+description: How we recommend installing Python on Windows, and how to use Python Windows features
+icon: fab fa-windows
+toc: 3
+---
+
+Our recommended way of installing Python on a Windows operating system is using the full installer from the official
+[python.org Downloads page](https://www.python.org/downloads/) (hit the big yellow button) using the default options,
+except making sure to check the "Add python.exe to PATH" checkbox. Getting Python from the [Microsoft
+Store](https://apps.microsoft.com/store/search/python) is _not_ recommended as [it can cause various
+issues](../microsoft-store).
+
+This guide gives detailed instructions on that [recommended way to install Python on Windows](#recommended-install),
+then goes on to discuss [more information about installing Python on Windows](#more-installation-information), and
+finally wraps up by explaining some common Windows-specific Python usage, namely the [py launcher](#the-py-launcher),
+and how to work with [virtual environments](#virtual-environments).
+
+You may also want to check out our guides on [Common Issues Using Python on Windows](../common-issues), [Adding Python
+to the Windows Path](../putting-python-on-path.md), and [Setting up a Unix-Style Environment on
+Windows](../unix-env-on-windows.md).
+
+## Recommended Install
+
+Follow the five steps below to install the latest version of Python on Windows.
+
+(The instructions were written with Windows 10 and Python 3.11.0 in mind, but should be nearly or fully identical with
```

Comment:
I'd prefer to mention it. Lots of times I see tutorials that don't mention the software versions involved or date of writing and I have to guess if it'll work for my setup.

Mentioning Win 11 might help people from clicking away thinking "oh this must be a Win 10 guide since Win 11 is new".

(Though btw I have not explicitly tested the guides in Win 11 since I'm still on Win 10.)

Reply:
Fair, fwiw i'm on windows 11 and everything seems to be the same.